internal/pkg/service/buffer/task/orchestrator/orchestrator.go

```diff
@@ -131,19 +127,19 @@ func Start[T any](ctx context.Context, wg *sync.WaitGroup, d dependencies, confi
 	if err := config.Validate(); err != nil {
 		panic(err)
 	}
+
 	w := &orchestrator[T]{
-		logger:       d.Logger().AddPrefix(fmt.Sprintf("[orchestrator][%s]", config.Name)),
-		tracer:       d.Tracer(),
-		client:       d.EtcdClient(),
-		dist:         d.DistributionWorkerNode(),
-		tasks:        d.TaskNode(),
-		config:       config,
-		allowedTypes: make(map[etcdop.EventType]bool),
-	}
-	for _, eventType := range config.Source.WatchEvents {
-		w.allowedTypes[eventType] = true
+		logger: d.Logger().AddPrefix(fmt.Sprintf("[orchestrator][%s]", config.Name)),
+		tracer: d.Tracer(),
+		client: d.EtcdClient(),
+		dist:   d.DistributionWorkerNode(),
+		tasks:  d.TaskNode(),
+		config: config,
 	}
 
+	// Delete events are not needed/ignored
+	w.config.Source.WatchEtcdOps = append(w.config.Source.WatchEtcdOps, etcd.WithFilterDelete())
+
```

Comment:
Nepotrebujeme sledovat `delete` events, zbytocne nam to server posle, ked sme to aj tak zahadzovali.